<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grov::Main_window</class>
 <widget class="QMainWindow" name="grov::Main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google Reader Offline Viewer</string>
  </property>
  <widget class="QWidget" name="parent_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="grov::main_window::Viewer" name="viewer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="go_offline"/>
    <addaction name="discard_all_offline_data"/>
    <addaction name="flush_offline_data"/>
    <addaction name="quit"/>
   </widget>
   <widget class="QMenu" name="feed_menu">
    <property name="title">
     <string>&amp;Feed</string>
    </property>
    <addaction name="previous_item"/>
    <addaction name="next_item"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="item_menu">
    <property name="title">
     <string>&amp;Item</string>
    </property>
    <addaction name="go_to_item_page"/>
    <addaction name="star"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="feed_menu"/>
   <addaction name="item_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="go_offline">
   <property name="text">
    <string>Go &amp;offline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="flush_offline_data">
   <property name="text">
    <string>&amp;Flush offline data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="next_item">
   <property name="text">
    <string>&amp;Next item</string>
   </property>
   <property name="shortcut">
    <string>J</string>
   </property>
  </action>
  <action name="previous_item">
   <property name="text">
    <string>&amp;Previous item</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="discard_all_offline_data">
   <property name="text">
    <string>&amp;Discard all offline data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="go_to_item_page">
   <property name="text">
    <string>&amp;Go to item's page</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="star">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Star</string>
   </property>
   <property name="toolTip">
    <string>Star</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>grov::main_window::Viewer</class>
   <extends>QWidget</extends>
   <header location="global">grov/main_window/viewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>activated()</signal>
   <receiver>grov::Main_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
